app/models/scheme.rb

```diff
@@ -0,0 +1,7 @@
+class Scheme < ApplicationRecord
+  belongs_to :organisation
```

Comment:
Also worth adding

`has_many: :schemes` to `Organisation I think? (So you can do `organisation.schemes`)

Reply:
I had it earlier but removed, since didnt need it excplicitely in this case - yagni. But will will likely need it at some point

Reply:
Will do